Src/Base/AMReX_MultiFab.cpp

```diff
@@ -167,6 +167,27 @@ MultiFab::Add (MultiFab& dst, const MultiFab& src,
     amrex::Add(dst, src, srccomp, dstcomp, numcomp, nghost);
 }
 
+MultiFab
+MultiFab::copy ()
+{
+    auto mf = MultiFab(
+        this->boxArray(),
+        this->DistributionMap(),
+        this->nComp(),
+        this->nGrowVect(),
+        MFInfo().SetArena(this->arena())
```

Comment:
```suggestion
        MFInfo().SetArena(this->arena()),
        this->Factory()
```
Use the same factory too. For EB, the new MultiFab will contains EB information.

Reply:
Ah, that's good to know. Adding to https://github.com/AMReX-Codes/pyamrex/pull/282, too